obu/.clang-format

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Move the clang-format config file to `ROOT_DIR`. The same reason for `.gitignore`.